arPullGlob.py

```diff
@@ -507,12 +623,13 @@ def XfelDsply(strt):
     zzz=[]
     totalLo=[]
     totalHi=[]
+    BEGIN=datetime.dateime(2022,3,25,8,0,0)
     #print(matplotlib.__version__, numpy.__version__)
     #e,f,DisLow,DisHi, DisLoStd, DisHiStd = CmStats(datetime.time,datetime.time)
     #print "e=", e  , "\n\r", "DisLow=", DisLow, "DisLoStd=",DisLoStd
     #e, f, r1,r2,r3,r4  = cavStats("4","27",startDate,endDate)
     #print ("e=",e,"r1=", r1, r2) 
-    squid = XfelDsply(startDate)
+    squid = XfelDsply(BEGIN,startDate)
```

Comment:
The variable names pain me...

Reply:
I know. slowly but surely I'll rename them.